privacyidea/lib/tokenclass.py

```diff
@@ -1497,10 +1492,11 @@ def check_last_auth_newer(self, last_auth):
                       "tdelat %s: %s" % (self.token.serial, tdelta,
                                          last_success_auth))
             # convert string of last_success_auth
-            last_success_auth = datetime.datetime.strptime(
-                last_success_auth, AUTH_DATE_FORMAT)
+            #last_success_auth = datetime.datetime.strptime(
+            #    last_success_auth, AUTH_DATE_FORMAT)
+            last_success_auth = parse_date_string(last_success_auth)
             # The last auth is to far in the past
-            if last_success_auth + tdelta < datetime.datetime.now():
+            if last_success_auth + tdelta < datetime.datetime.utcnow():
```

Comment:
Right now, last auth is stored without timezone information in the database.
I'm wondering if it would be better to store it with timezone information in order to have a uniform format for datetime values in token info values? If we want to do that, we need to change the right side to an aware datetime object.